tests/fixtures/Page.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\CMS\Controllers\ContentController;
+
+class Page extends SiteTree
```

Comment:
Isn't this going to cause issues when `mysite` is installed with another Page.php? It'll also cause additional IDE and code inspection issues.

Suggested change: Rename to non-php extension, and perform a file copy in bootstrap.php if it detects no `$project` folder.